qrest/conf.py

```diff
@@ -153,6 +191,14 @@ def _validate(self):
             raise RestClientConfigurationError(
                 "parameter 'exclusion_group' should be 'None' for FileParameter"
             )
+        if self.example:
```

Comment:
If `self.example == []`, lines 195-197 aren't executed and looking at the exception message, they should be.